packages/eth-edge/src/services/transactions.service.ts

```diff
@@ -52,4 +55,12 @@ export class TransactionsService {
         // has already been processed by the system
         await this.transactionsRepository.save(transactions);
     }
+
+    async findAll(userId: number): Promise<Transaction[]> {
+        const userWallets = await this.walletsService.findAll(userId);
+        const addresses = userWallets.map((t) => t.address.toLowerCase());
```

Comment:
We should make sure to also lower case all addresses in general when we insert?
```suggestion
        const addresses = userWallets.map((transaction) => transaction.address.toLowerCase());
```